### PR TITLE
[ADD] support_branding

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by OSguard** [comment]
:+1:

File: support_branding/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    ],
+    "qweb": [
+        'static/src/xml/base.xml',
+    ],
+    "data": [
+        "data/ir_config_parameter.xml",
+        'views/qweb.xml',
+    ],
+    'active': True,
+}
```
Comment (StefanRijnhart) [inline support_branding/__openerp__.py:39]: license key missing

Comment (hbrunn) [inline support_branding/__openerp__.py:39]: @StefanRijnhart done

File: support_branding/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        'web',
+    ],
+    "qweb": [
+        'static/src/xml/base.xml',
+    ],
+    "data": [
+        "data/ir_config_parameter.xml",
+        'views/qweb.xml',
+    ],
+    'active': True,
```
Comment (tafaRU) [inline support_branding/__openerp__.py]: Please replace it by `auto_install` since this key is obsolete.

Comment (hbrunn) [inline support_branding/__openerp__.py]: @tafaRU good catch! This is a remainder from when the module was in our private repo, I think it shouldn't be autoinstall here. So I removed the key entirely.

**Comment by pedrobaeza** [comment]
Very cool module! Tested and :+1:

**Comment by OSguard** [comment]
:+1:

**Comment by StefanRijnhart** [comment]
:+1:, thanks!

**Pull request merged**